src/main/java/nl/dtls/fairdatapoint/service/impl/FairMetaDataMetricsServiceImpl.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package nl.dtls.fairdatapoint.service.impl;
+
+import com.google.common.base.Preconditions;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
+import nl.dtl.fairmetadata4j.model.Metric;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service layer for fair metric
+ *
+ * @author Rajaram Kaliyaperumal <rr.kaliyaperumal@gmail.com>
+ * @author Kees Burger <kees.burger@dtls.nl>
+ * @since 2018-01-16
+ * @version 0.1
+ */
+@Service("fairMetaDataMetricsServiceImpl")
+public class FairMetaDataMetricsServiceImpl {
```

Comment:
The `@Service` stereotype is not really used, we can fall back to `@Component`. Defining the name is not required either, it is derived from the class name.

Reply:
Is this applicable for all service layer?

Reply:
In our case: almost always yes.